src/plugins/Makefile.am

```diff
@@ -221,7 +223,31 @@ libgs_plugin_flatpak_la_SOURCES =			\
 libgs_plugin_flatpak_la_LIBADD = $(GS_PLUGIN_LIBS) $(FLATPAK_LIBS)
 libgs_plugin_flatpak_la_LDFLAGS = -module -avoid-version
 libgs_plugin_flatpak_la_CFLAGS = $(GS_PLUGIN_CFLAGS) $(WARN_CFLAGS)
-endif
+
+libgs_plugin_legacy_external_apps_la_SOURCES =	\
+	gs-appstream.c				\
+	gs-appstream.h				\
+	gs-flatpak.c				\
+	gs-flatpak.h				\
+	gs-flatpak-symlinks.c			\
+	gs-flatpak-symlinks.h			\
+	gs-plugin-legacy-external-apps.c
+libgs_plugin_legacy_external_apps_la_LIBADD = $(GS_PLUGIN_LIBS) $(FLATPAK_LIBS) $(JSON_GLIB_LIBS)
+libgs_plugin_legacy_external_apps_la_LDFLAGS = -module -avoid-version
+libgs_plugin_legacy_external_apps_la_CFLAGS = $(GS_PLUGIN_CFLAGS) $(JSON_GLIB_CFLAGS) $(WARN_CFLAGS)
+
+libgs_plugin_external_runtimes_cleaner_la_SOURCES =		\
```

Comment:
Can you add the word "legacy" also in the name of this plugin? To make it clear it applies to the same legacy ext apps as the other plugin.